### PR TITLE
Fix issue#30, exclude 'g++', 'c++' and 'i++' from getting karma

### PR DESCRIPTION
**Pull request by bitlord**

### CODE REVIEW

File: plugin.py [hunk 1]
```diff
@@ -723,6 +723,10 @@ def _do_karma(self, irc, channel, agent, recip, line, explicit=False):
         # Extract 'puiterwijk' out of 'have a cookie puiterwijk++'
         recip = recip.strip().split()[-1]
 
+        # Exclude 'c++', 'g++' or 'i++' (c,g,i), issue #30
+        if len(recip) == 1 and recip.lower() in ['c','g','i']:
```
Comment (pypingou) [inline plugin.py]: If the first part ``len(recip) == 1`` needed?

At worst to ensure it is not ``None`` you could simply do: ``str(recip).lower() in ...`` :)

File: plugin.py [hunk 2]
```diff
@@ -834,7 +838,7 @@ def mirroradmins(self, irc, msg, args, hostname):
                "mirroradmins?name=" + hostname)
         result = self._load_json(url)
         if not 'admins' in result:
-            irc.reply(result.get('message', 'Something went wrong')
+            irc.reply(result.get('message', 'Something went wrong'))
```
Comment (pypingou) [inline plugin.py]: Good catch ;-)

File: plugin.py (hunk 1 above)
Comment (bitlord) [inline plugin.py]: @pypingou 
> If the first part len(recip) == 1 needed?
I think it is (maybe I'm wrong), we need to be sure that the recipient/user/nick is exactly one letter long, and those letters are 'c','g' or 'i'?

Comment (bitlord) [inline plugin.py]: I was rethinking about what you said, and I understand now!
it compares whole string **recip** in the list, not the first letter ..., so ``len(recip) == 1`` is unneeded.

Comment (bitlord) [inline plugin.py]: Thanks @pypingou I understand all suggestions, I tested it with my script, and learned something ;-)
Also I updated the branch with your suggestions!

**Comment by pypingou** [comment]
:+1: for me, thanks!

**Pull request merged**